src/components/multi-action-button/multi-action-button.pw.tsx

```diff
@@ -152,38 +145,6 @@ test.describe("Prop tests", () => {
     await expect(actionButton).toHaveCSS("background-color", "rgb(0, 77, 42)");
   });
 
-  test(`should render component in a hidden container`, async ({
```

Comment:
Removed this test so we have previously covered the rendering of the child buttons when the main one is hovered